partials/author-bio-social-links.php

```diff
@@ -23,25 +23,25 @@
 <ul class="social-links">
 	<?php if ( $fb = $author_obj->fb ) { ?>
 		<li class="facebook">
-			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="<?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
+			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="Follow <?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
 		</li>
 	<?php } ?>
 
 	<?php if ( $twitter = $author_obj->twitter ) { ?>
 		<li class="twitter">
-			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>"><i class="icon-twitter"></i></a>
+			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>" title="Follow <?php echo largo_twitter_url_to_username ( $twitter ); ?> on Twitter" ><i class="icon-twitter"></i></a>
```

Comment:
```diff
- echo largo_twitter_url_to_username ( $twitter ); 
+ echo esc_attr( largo_twitter_url_to_username( $twitter ) ); 
```
The twitter username output needs to be passed through `esc_attr` for safety, and PHP does not permit spaces between a function name and the following `(`

Reply:
Hi @benlk , comments are addressed